src/components/ui/emoji-picker/EmojiPicker.module.scss

```
@@ -196,8 +226,56 @@
+  }
+
+  .starOverlay {
+    position: absolute;
+    top: 4px;
+    right: 4px;
+    z-index: 10;
+    opacity: 0;
+    transition: all 0.2s ease;
+    
+    color: var(--primary-color); 
+    background: rgba(30, 30, 30, 0.8);
```

Stylelint (17.2.0)

[error] Expected "0.8" to be "80%" (alpha-value-notation)

(alpha-value-notation)